test/unit/BondingManager.js

```diff
@@ -1339,6 +1577,24 @@ contract("BondingManager", accounts => {
             assert.equal(endTotalStake.sub(startTotalStake), 1000, "should update transcoder's total stake in the pool with new rewards")
             assert.equal(endTotalBonded.sub(startTotalBonded), 1000, "should update total bonded with new rewards")
         })
+
+        it("should update caller with rewards if lastActiveStakeUdpateRound < currentRound", async () => {
```

Comment:
"lastActiveStakeUdpateRound" -> "lastActiveStakeUpdateRound"

Reply:
fixed in 445b1a3